crates/oxc_linter/src/snapshots/no_self_compare.snap

```diff
@@ -0,0 +1,12 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 53
+expression: no_self_compare
+---
+
+  × eslint(no-self-compare): Disallow comparisons where both sides are exactly the same
+   ╭─[no_self_compare.tsx:1:1]
+ 1 │ x === x
+   · ───────
```

Comment:
It would be nice to point to the two expressions

Reply:
There is a `GetSpan` trait implemented by all Ast types which does the matching for you. You can bring in that trait and just call `span()` on the expression.

Reply:
Thank you!